openssl2/src/extension.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+/// Extended Key Usage.
+#[derive(Debug)]
+pub struct ExtKeyUsage {
+    /// Server authentication.
+    server_auth: bool,
+    // extKeyUsage contains other fields, but only serverAuth is used in Certs Service.
+    // Unused other fields are omitted.
+}
+
+impl ExtKeyUsage {
+    /// Parse an extension as `extKeyUsage`.
```

Comment:
Might be useful to add the URLs of the RFCs and sections that document the structure.

Reply:
Fixed